setuptools_rust/extension.py

```diff
@@ -170,6 +170,16 @@ def get_rust_version(self) -> Optional[SimpleSpec]:  # type: ignore[no-any-unimp
                 "Can not parse rust compiler version: %s", self.rust_version
             )
 
+    def get_cargo_profile(self) -> Optional[str]:
+        args = self.args or []
+        try:
+            index = args.index("--profile")
+            return args[index + 1]
```

Comment:
The profile can also be passed as `--profile=custom`. Is it worth it trying to handle this?

Reply:
Thanks for the review, fixed.